services/CollectionService/src/index.php

```diff
@@ -22,23 +23,12 @@
   'twig.path' => __DIR__.'/../templates',
 ));
 
-$islandoraResourceServiceProvider = new \Islandora\ResourceService\Provider\ResourceServiceProvider;
```

Comment:
Sorry, some how lost here. How does the collection.controller makes use of the resource routes without having them registered or mounted? Who is mounting them? Thanks!